tests/datasets/test_oscd.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import glob
+import os
+import shutil
+from pathlib import Path
+from typing import Generator
+
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.monkeypatch import MonkeyPatch
+from torch.utils.data import ConcatDataset
+
+import torchgeo.datasets.utils
+from torchgeo.datasets import OSCD
+
+
+def download_url(url: str, root: str, *args: str, **kwargs: str) -> None:
+    shutil.copy(url, root)
+
+
+class TestOSCD:
+    @pytest.fixture
+    def dataset(
+        self, monkeypatch: Generator[MonkeyPatch, None, None], tmp_path: Path
+    ) -> OSCD:
+        monkeypatch.setattr(  # type: ignore[attr-defined]
+            torchgeo.datasets.oscd, "download_url", download_url
+        )
+        # idk if this is right yet
```

Comment:
Comment should be removed if md5 is correct (you can test this with `md5sum OSCD.zip`)